addon/registry.js

```diff
@@ -0,0 +1,57 @@
+let REGISTERED_RECEIVERS = {};
+
+export function registerReceiver(component) {
+  let componentName = getComponentRegistryKey(component);
+  REGISTERED_RECEIVERS[componentName] = component;
```

Comment:
It currently does overwrite it. 👍 on raising an assertion, we can instruct to use a custom `sendToName`